README.md

```diff
@@ -1,236 +1,130 @@
 <a name="readme-top"></a>
 
```

Comment:
- [ ] [REQUIRED] I noticed that you still don't have a video presentation with one requierment for this milestone . Please include it as already mentioned by previous reviewer 👍